File: backend/src/main/java/sb/backend/admin/modelo/VentaPasaje.java
package sb.backend.admin.modelo;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;
import jakarta.persistence.*;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class VentaPasaje {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer idVenta;

    private String origen;
    private String destino;
    private String fechaViaje;
    private String horaPartida;
    private String horaLlegada;
    private String asiento;

    private String tipoDocumento;
    private String numeroDocumento;
    private String nombres;
    private String apellidos;
    private String fechaNacimiento;
    private String genero;

    private Double precio;

    @ManyToOne
    @JoinColumn(name = "idItinerario")
    private Itinerario itinerario;

}

File: front-SistemaWeb/src/app/core/model/ventaPasaje.ts
import { Itinerario } from "./itinerario";

export class VentaPasaje {
    idVenta: number;
    origen: string;
    destino: string;
    fechaViaje: string;
    horaPartida: string;
    horaLlegada: string;
    asiento: string;
    tipoDocumento: string;
    numeroDocumento: string;
    nombres: string;
    apellidos: string;
    fechaNacimiento: string;
    genero: string;
    precio: number;
    itinerario: Itinerario;
}